hedera-node/src/main/java/com/hedera/services/store/tokens/unique/UniqueTokenStore.java

```diff
@@ -56,6 +62,7 @@
 public class UniqueTokenStore extends BaseTokenStore implements UniqueStore {
 
 	private final Supplier<FCInvertibleHashMap<MerkleUniqueTokenId, MerkleUniqueToken, OwnerIdentifier>> uniqueTokenSupplier;
+	private final List<Long> lastMintedSerialNumbers;
```

Comment:
We won't need this if we refactor the `mint` to use `CreationResult` instead of `ResponseCode`.